File: python_examples/gnu_octave_to_python/Chapman (MatLab Programming for Engineers)/Chapter 3- Branching statements, Program design/body_temperature.m
%prompt user for input
temp = input('Enter body temperature (in fahrenheit):');

if temp <= 97.5
  fprintf('Body temperature below normal\n');
elseif (temp > 97.5) & (temp <= 99.5)
  fprintf('Body temperature normal\n');
elseif (temp > 99.5) & (temp <= 103.0)
  fprintf('Body temperature slightly high\n');
else
  fprintf('Body temperature dangerously high\n');
endif

File: python_examples/body_temperature.py
#prompt user for input
temp = float(input('Enter body temperature (in fahrenheit):'))

if temp <= 97.5:
    print('Body temperature below normal')
elif (temp > 97.5) & (temp <= 99.5):
    print('Body temperature normal')
elif (temp > 99.5) & (temp <= 103.0):
    print('Body temperature slightly high')
else:
    print('Body temperature dangerously high')